Update llm-gateway-redaction.mdx (#5077)
langchain-ai/docs: src/langsmith/llm-gateway-redaction.mdx

Detection uses Presidio for named entities (names, locations, and NRP) and pattern-based rules for structured identifiers.

> Other forms are pattern detection do not use a model and area able to identify it with regular expressions: 

Reviewer: ```suggestion
Structured identifiers are detected with regular expressions and do not use a model:

```
